src/test/velodromeCL.test.ts

```diff
@@ -182,6 +182,22 @@ const testVelodromeCL = ({ wallet, network, provider }: TestingRunParams) => {
         );
       });
 
+      it("increases liquidity in a  staked CL position", async () => {
+        const usdcBalance = await pool.utils.getBalance(USDC, pool.address);
+        const wethBalance = await pool.utils.getBalance(WETH, pool.address);
+        const positionBefore = await velodromePositionManager.positions(
+          tokenId
+        );
+        await pool.increaseLiquidity(
+          Dapp.VELODROMECL,
+          tokenId,
+          usdcBalance.div(2),
+          wethBalance.div(2)
+        );
+        const positionAfter = await velodromePositionManager.positions(tokenId);
+        expect(positionAfter.liquidity.gt(positionBefore.liquidity));
+      });
+
```

Comment:
this test is not used any more. there is no onchain function to increaseLp for a staked position.

Reply:
removed it and added tests for Aerodorme CL. I had to use the USDC-USDT pair because the gauge for  USDC-WETH is not enabled on contract side. [4bf54a6](https://github.com/dhedge/dhedge-v2-sdk/pull/60/commits/4bf54a67fd05e302e981af0a7822298b75c4f2a3)